### PR TITLE
Host external assets locally

### PR DESCRIPTION
**Pull request by shadyvb**
Ref https://github.com/humanmade/product-dev/issues/886

### CODE REVIEW

File: inc/ui/namespace.php [hunk 1]
```diff
@@ -63,9 +63,9 @@ function admin_bar_menu( WP_Admin_Bar $wp_admin_bar ) {
  * We enqueue all the scripts and styles for the documentation page here.
  */
 function load_page() {
-	wp_enqueue_style( 'highlightjs', 'https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.6/styles/vs2015.min.css' );
-	wp_enqueue_script( 'highlightjs', 'https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.6/highlight.min.js' );
-	wp_enqueue_script( 'highlightjs-line-numbers', 'https://cdn.jsdelivr.net/npm/highlightjs-line-numbers.js@2.7.0/dist/highlightjs-line-numbers.min.js' );
+	wp_enqueue_style( 'highlightjs', plugins_url( '/assets/vs2015.min.css', Documentation\DIRECTORY . '/wp-is-dumb' ) );
```
Comment (roborourke) [inline inc/ui/namespace.php:66]: Forgot about the wp-is-dumb thing 😂

**Review by roborourke — APPROVED** [review]

**Comment by roborourke** [comment]
Hmm, ideally we can tell HM Linter to ignore those minified assets, may need a `.phpcsignore` adding.

**Comment by shadyvb** [comment]
The CS stuff are ongoing issues, so will merge this here, can tackle after discussion / feedback.

**Comment by roborourke** [comment]
Not sure what you mean there, regarding stylelint ignore support? This PR introduced the failing check but if there's no way to fix it without an update to HM Linter then it makes sense.

**Comment by shadyvb** [comment]
> This PR introduced the failing check but if there's no way to fix it without an update to HM Linter then it makes sense.

My comment was based on yours, assumed local ignore directions wouldn't be respected by the bot. Worth revisiting, I think we're seeing this in multiple locations ( can't remember where else right now ).

**Comment by roborourke** [comment]
It's only the analytics plugin as far as I'm aware but that's because of the CSS copied in from the accelerate plugin.

The HM Linter readme states that the ignore files should be respected so maybe it is a bug upstream.

**Pull request merged**